fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateRoutineLoadStmt.java

```diff
@@ -118,6 +119,7 @@ public class CreateRoutineLoadStmt extends DdlStmt {
     public static final String TRIMSPACE = "trim_space";
     public static final String ENCLOSE = "enclose";
     public static final String ESCAPE = "escape";
+    public static final String FILL_MISMATCH_COLUMN_WITH = "fill_mismatch_column_with";
```

Comment:
the same as `TableFunctionTable`.
put this property and MisMatchFillValue in `LoadStmt`?

Reply:
LoadStmt is only for broker load, the current implementation is to define fields in respective stmt